mate-panel/panel-toplevel.c

```diff
@@ -505,7 +505,7 @@ static void panel_toplevel_begin_grab_op(PanelToplevel* toplevel, PanelGrabOpTyp
 	widget = GTK_WIDGET (toplevel);
 	window = gtk_widget_get_window (widget);
 
-	toplevel->priv->grab_op          = op_type;
+	toplevel->priv->grab_op          = (op_type != FALSE);
 	toplevel->priv->grab_is_keyboard = grab_keyboard;
```

Comment:
It seems that this change doesn' t fix this warning.
```
panel-toplevel.c:509:37: warning: implicit conversion changes signedness: 'gboolean' (aka 'int') to 'guint' (aka 'unsigned int') [-Wsign-conversion]
```

Reply:
The warning is on `grab_keyboard`.

```
panel-toplevel.c:509:37: warning: implicit conversion changes signedness: 'gboolean' (aka 'int') to 'guint' (aka 'unsigned int') [-Wsign-conversion]
        toplevel->priv->grab_is_keyboard = grab_keyboard;
                                         ~ ^~~~~~~~~~~~~
```

Reply:
done

Reply:
yeap, warning is gone.